Modules/Common/src/MeanIsAbove.cxx

```diff
@@ -57,14 +55,7 @@ Quality MeanIsAbove::check(std::map<std::string, std::shared_ptr<MonitorObject>>
 void MeanIsAbove::beautify(std::shared_ptr<MonitorObject> mo, Quality checkResult)
 {
   // A line is drawn at the level of the threshold.
-  // Its colour depends on the quality.
-
-  if (!this->isObjectCheckable(mo)) {
```

Comment:
This is a change in behaviour. There are similar uses removed in TOF code. I think we should not break the existing behaviour that users might rely on.

Extract a helper which performs a similar check, but is called `encapsulatedInheritFrom(std::string_view)` and is a public member of MonitorObject. Find all the former uses of `isObjectCheckable` and use the new helper there instead.